components/wc-data/wc-data.html

```diff
@@ -0,0 +1,46 @@
+<script src="../../dist/winston-churchill.js"></script>
+<template></template>
+<script>
+var dm = WC.register('wc-data'),
+
+    goGetIt = function (src, payload, cb) {
+        if (!src) { return; }
+        var xhr = new XMLHttpRequest();
```

Comment:
You're right. That's not fetch. I did the wc-data component before I did the polyfill. Haven't gotten around to updating stuff yet.